Zend/zend_portability.h

```diff
@@ -62,6 +62,23 @@
 
 #include <limits.h>
 
+#if HAVE_SYS_PARAM_H
+# include <sys/param.h>
+#endif
+
+#ifndef MAXPATHLEN
+# if _WIN32
+#  include "win32/ioutil.h"
+#  define MAXPATHLEN PHP_WIN32_IOUTIL_MAXPATHLEN
+# elif PATH_MAX
+#  define MAXPATHLEN PATH_MAX
+# elif defined(MAX_PATH)
+#  define MAXPATHLEN MAX_PATH
+# else
+#  define MAXPATHLEN 256
+# endif
+#endif
```

Comment:
This looks out of place for this header. Can it be moved into zend_virtual_cwd.h?

Reply:
It can be moved there, yes... There is only place this particular one is needed from this tsrm config common header. I have here also next step in mind - to somehow sync the chuck of very similar code in main/php.h and this one but after this pull request so we won't break something too soon. :)

Reply:
Having it in zend_virtual_cwd.h should be fine for that, as it is included from php.h.

Reply:
Moved to zend_virtual_cwd.h in the next commit(s).